checkST2110.js

```diff
@@ -473,35 +580,50 @@ const extractMTParams = (sdp, params = {}) => {
   let isSkippedType = false;
   let streamCount = 0;
   let payloadType = -1;
-  for ( let x = 0 ; x < lines.length ; x++ ) {
+  let mediaType = -1;
+  let encodingName = -1;
+
+  for (let x = 0; x < lines.length; x++) {
```

Comment:
`extractMTParams` could do with refactoring... it probably ought to split on `m=` and then produce one entry for each media description, whereas currently files with e.g. two `a=fmtp:` lines per media description, or none (some media types don't need one), or media descriptions where the `a=rtpmap:` line comes after the `a=fmtp:` line rather than befeore, will produce unexpected results...

But since it had all these problems before this PR, let's save that refactoring for another time!

Reply:
Agreed

Reply:
Created and issue to capture this https://github.com/AMWA-TV/sdpoker/issues/26